recipe/meta.yaml

```diff
@@ -1,22 +1,33 @@
 {% set unix_version = "5.32.0" %}
 {% set win_version = unix_version + ".1" %}
+{% set major_minor_version = unix_version.split(".")[:2] | join(".") %}
 
 package:
   name: perl
   version: {{ unix_version }}  # [not win]
   version: {{ win_version }}   # [win]
 
 source:
-  url: http://www.cpan.org/src/5.0/perl-{{ unix_version }}.tar.gz                                                 # [unix]
-  url: http://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-64bit-portable.zip  # [win64]
-  url: http://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-32bit-portable.zip  # [win32]
-  sha256: efeb1ce1f10824190ad1cadbcccf6fdb8a5d37007d0100d2d9ae5f2b5900c0b4                                        # [unix]
-  # sha1 hashes published at: http://strawberryperl.com/releases.html
-  sha1: 25cc36aa98e5ec6a5d507a2a7a6e39eb655795f3                                                                  # [win64]
-  sha1: 61b204137c6c4277b5155eb5cc0514e9e4674625                                                                  # [win32]
+  - url: http://www.cpan.org/src/5.0/perl-{{ unix_version }}.tar.gz                                                 # [unix]
+    sha256: efeb1ce1f10824190ad1cadbcccf6fdb8a5d37007d0100d2d9ae5f2b5900c0b4                                        # [unix]
+    patches:                                                                                                        # [unix]
+      - 0001-Use-userelocatableinc-together-with-useshrplib.patch                                                   # [unix]
+  - url: http://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-64bit-portable.zip  # [win64]
+    url: http://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-32bit-portable.zip  # [win32]
+    sha1: 25cc36aa98e5ec6a5d507a2a7a6e39eb655795f3                                                                  # [win64]
+    sha1: 61b204137c6c4277b5155eb5cc0514e9e4674625                                                                  # [win32]
+    # sha1 hashes published at: http://strawberryperl.com/releases.html
 
 build:
-  number: 0
+  number: 1
+  string: {{ PKG_BUILDNUM }}_h{{ PKG_HASH }}_strawberry  # [win]
+  string: {{ PKG_BUILDNUM }}_h{{ PKG_HASH }}_perl5       # [not win]
+  run_exports:
+    weak:
+      - perl {{ major_minor_version }}.* *_strawberry  # [win]
+      - perl {{ major_minor_version }}.* *_perl5       # [not win]
+    noarch:                                            # [not win]
+      - perl                                           # [not win]
```

Comment:
(I just noticed that I forgot to commit/push this change before, whoops.)

Reply:
I think, ideally we'd rather have a `perl_abi`/`perl5_abi`/??? or something package which we should use instead of relying on build string matching here. E.g., have a `perl5_abi {{ major_minor_version }}.*` weak export.
I haven't worked on this lately and thus not thought it completely through.
Suggestions and reviews are welcome!

Reply:
Using glob-style build string matching is fine for now (we already use this style throughout conda-forge, so it's nothing scandalous). If someone wants to refine this later on, that'd be fine and should still be possible.